type-definitions/ts-tests/from-js.ts

```diff
@@ -33,13 +33,13 @@ test('fromJS', () => {
 });
 
 test('fromJS in an array of function', () => {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  /* eslint-disable @typescript-eslint/no-explicit-any */
   const create = [(data: any) => data, fromJS][1];
 
-  expect(create({ a: 'A' })).type.toBeAny();
+  expect(create({ a: 'A' })).type.toBe<any>();
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const createConst = ([(data: any) => data, fromJS] as const)[1];
+  /* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
This seems weird 🤔

Reply:
I though two comment lines are less than three (;

This is the same like `// eslint-disable-next-line` would be used on every line between `/* eslint-disable */` and `/* eslint-enable */`. That’s just another syntax. I can bring back `// eslint-disable-next-line` if you prefer.

Reply:
I prefer the `next-line` version because it does not affect a whole range that can grow bigger in the future.